content/blogposts/2022/release-3.42.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Sourcegraph 3.42 release"
+publishDate: 2022-07-22T10:00-07:00
+description: "Sourcegraph 3.42 introduces new admin analytics, a new search UI in Beta, alternate search suggestions, and upgrades for code insights running on repo lists and monorepos."
```

Comment:
@iskyOS - Should Code Insights be capitalized here?

Reply:
Yes! Good catch — saw this earlier but forgot to change.

Reply:
```suggestion
description: "Sourcegraph 3.42 introduces new admin analytics, a new search UI in Beta, alternate search suggestions, and upgrades for Code Insights running on repo lists and monorepos."
```